monitoring-support/src/main/java/org/terracotta/monitoring/PlatformClientFetchedEntity.java

```diff
@@ -36,13 +35,13 @@
 
   public String clientIdentifier;
   public String entityIdentifier;
-  public transient ClientDescriptor clientDescriptor;
+  public transient Object clientDescriptor;
```

Comment:
@mathieucarbou : can you check to make sure you don't use the client ClientDescriptor here by direct access?